### PR TITLE
Review microverse trial

### PR DESCRIPTION
**Pull request by Booze33**
Create a pull request for code review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+         <ul>
+            <li>1914 - Born in Cresco, Iowa</li>
+            <li>1933 - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
+            <li>1935 - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+            <li>1937 - Finishes university and takes a job in the US Forestry Service</li>
+            <li>1938 - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
+            <li>1941 - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science.</li>
+            <li>1942 - Receives a Ph.D. in Genetics and Plant Pathology</li>
+            <li>2009 - dies at the age of 95.</li>
```
Comment (Booze33) [inline index.html:30]: I suggest you align your list. It will look nice if your list starts at the same point.  🔢

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+         </ul>
+         </div>
+         <p>If you have time, you should read more about this incredible human being on his <a id="tribute-link"  href="https://en.wikipedia.org/wiki/Norman_Borlaug">Wikipedia entry.</a></p>
```
Comment (Booze33) [inline index.html:33]: I suggest this should be outside the #main. It will be better off with its own styling because it kind of looks like it is part of the list giving it its own style will make the page nicer. 💯

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+#main {
+  background-color: rgb(184, 183, 183);
+}
+
+#div {
+  background-color: white;
+}
+
+#body {
```
Comment (Booze33) [inline styles.css:20]: I'd like to kindly suggest that the styling should be done more carefully. At the moment it looks incomplete.

**Review by Booze33 — COMMENTED** [review]
Good job! 
Need to work on the changes required.
Need to add linters to check the errors.

File: styles.css (hunk 3 above)
Comment (Favourezeugwa) [reply styles.css:20]: - I think it will be better to use polite suggestion here instead of an order.  You can go in the lines of  `I'd like to kindly suggest that the styling should be done more carefully. . . . .` 🤗

File: index.html (hunk 2 above)
Comment (Favourezeugwa) [reply index.html:33]: - I'd like to kindly suggest that you state reasons WHY this changes needs to be made. For example,  you can rephrase this to something like ` I suggest this should be outside the #main. It will be better of with it own styling because outside the main has .  . . . . . . . ` .
- Feep free to state a valid reason why you require a change. 😊

File: index.html (hunk 1 above)
Comment (Favourezeugwa) [reply index.html:30]: [OPTIONAL] :👍

- I think it will be great if you added emoji's to make your suggestions sound a bit more friendly. 🌸

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @Booze33 ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there. 

close lady

## Highlights 💯 
- Github review feature used ✔️
- Inline comments used✔️
- Some polite suggestions used✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- If you can, it will be great to add a GIF to your review summary.  Just like i did in this review summary on Required changes. 🌸

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Booze33 — COMMENTED** [review]
Changes required.
 Add your linters to detect errors for you.
Edit your READMe file.

**Comment by Booze33** [comment]
Hello,
  I made the changes on the review.

**Review by didierganthier — APPROVED** [review]
Hi @Booze33 ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Used inline comments ✔️ 
- Added emojis ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉
